docs/modules/user/pages/try/15_minutes.adoc

```diff
@@ -0,0 +1,229 @@
+= Try James in 15 minutes
+
+In this tutorial, we will set up a runing James demo using a prepared Docker image.
+We will then test the server by connecting with an email client. Finally, we
+will connect to the server via the REST-based Admin API.
+
+
+Requirements: 
+
+ * Docker
+ * curl
+ * Thunderbird client (optional)
+
+== Set up the demo server
+
+Before starting the server, because we will be connecting to the Admin API we need to set some configuration parameters.
+We will create a `webadmin.properties` file that James will use in order to allow us to connect to the API.
+
+Run this command to create the `webadmin.properties` file:
+
+[source,bash]
+----
+printf 'enabled=true\nport=8000\nhost=localhost' >> webadmin.properties
```

Comment:
Shouldn't we enable "webadmin" by default so that "override the configuration" step is no longer needed.

Note: with docker the port can be bound but not exposed

Reply:
That would be great, as it would indeed simplify the demo.

Small picky point, but would it also be possible to change the name of the image from "james-jpa-sample" to "james-demo"?

Reason: JPA is not relevant, and "demo" matches the documentation better.

Regarding the port, do you mean even without the `-p "8000:8000"` it is possible to connect to the docker image on port 8000? Or did you mean something else?

Reply:
> Small picky point, but would it also be possible to change the name of the image from "james-jpa-sample" to "james-demo"?

:+1: 

> Regarding the port, do you mean even without the -p "8000:8000" it is possible to connect to the docker image on port 8000? Or did you mean something else?

`-port 8000:8000` means that port 8000 of your container is bound with port 8000 of your computer.

We don't want people to disclose unauthenticated webadmin access by default for obvious security reasons.

We don't want people to do extra configuration because it is a pain.

So a good middle ground would be to enable always webadmin in docker images, and **warn** them that access to port 8000 should be limited for security reasons unless JWT authentication is setted up. 

Maybe we should document the port binding to loopback inerface to limit the risks.

Something like `docker run -p 127.0.0.1:8000:8000 ...` according to https://docs.docker.com/config/containers/container-networking/#published-ports. Maybe we might want to update existing README as well.

Also JWT authentication is not convenient enough. We don't have alternatives for webAdmin authentication and it is sad. I propose to add a mechanism to `None` and `Jwt` modes:  `Basic` using the `administratorId` and corresponding credentials as configured in `usersrepository.xml`. I will create a JIRA ticket for this.

Reply:
See https://issues.apache.org/jira/browse/JAMES-3186 ;-)

Reply:
I opened a few pull requests on that webadmin in docker topic:

 - https://github.com/apache/james-project/pull/195 Add webadmin enabled by default to jpa-guice docker image -> we can deliver this as part of 3.5.0 release IMO

 - https://github.com/linagora/james-project/pull/3387 Document webadmin port binding to loopback interface (just documentation)

Reply:
Updated exposure of port 8000 to be `-p "127.0.0.1:8000:8000"`.

Reply:
These will be captured as issues elsewhere, so I am resolving here.